torch/autograd/gradcheck.py

```diff
@@ -61,6 +61,24 @@ def get_numerical_jacobian(fn, input, target=None, eps=1e-3):
     x_tensors = iter_tensors(target, True)
     j_tensors = iter_tensors(jacobian)
 
+    def compute_gradient(x, idx, is_mkldnn=False):
+
+        def fn_out(is_mkldnn=False):
+            if not is_mkldnn:
+                return fn(input).clone()
```

Comment:
Is that expected that you just capture `input` from the parent scope here?

Reply:
yeah

Reply:
You might want to add a comment here stating that x is a view into input and so it just works.